MekHQ/src/mekhq/gui/model/UnitMarketTableModel.java

```diff
@@ -37,40 +42,49 @@
  * @author Neoancient
  */
 public class UnitMarketTableModel extends DataTableModel {
-	private static final long serialVersionUID = -6275443301484277495L;
+    //region Variable Declarations
+    private static final long serialVersionUID = -6275443301484277495L;
 
-	public static final int COL_MARKET = 0;
-	public static final int COL_UNITTYPE = 1;
-	public static final int COL_WEIGHTCLASS = 2;
-	public static final int COL_UNIT = 3;
-	public static final int COL_PRICE = 4;
-	public static final int COL_PERCENT = 5;
-	public static final int COL_NUM = 6;
+    public static final int COL_MARKET = 0;
+    public static final int COL_UNITTYPE = 1;
+    public static final int COL_WEIGHTCLASS = 2;
+    public static final int COL_UNIT = 3;
+    public static final int COL_PRICE = 4;
+    public static final int COL_PERCENT = 5;
+    public static final int COL_DELIVERY = 6;
+    public static final int COL_NUM = 7;
 
-	public UnitMarketTableModel() {
-        columnNames = new String[] { "Market", "Type", "Weight Class", "Unit", "Price", "Percent" };
-        data = new ArrayList<UnitMarket.MarketOffer>();
-	}
+    private final ResourceBundle resources = ResourceBundle.getBundle("mekhq.resources.GUI", new EncodeControl());
+    //endregion Variable Declarations
 
-    public int getColumnWidth(int c) {
-        switch (c) {
+    //region Constructors
+    public UnitMarketTableModel(final List<UnitMarketOffer> offers) {
+        columnNames = resources.getString("UnitMarketTableModel.columnNames").split(",");
+        setData(offers);
+    }
+    //endregion Constructors
+
+    public int getColumnWidth(final int column) {
+        switch (column) {
             case COL_MARKET:
-            case COL_UNIT:
-                return 100;
+            case COL_PRICE:
+                return 90;
             case COL_UNITTYPE:
+                return 15;
+            case COL_UNIT:
+                return 175;
             case COL_WEIGHTCLASS:
                 return 50;
-            case COL_PRICE:
-                return 70;
             default:
                 return 20;
         }
     }
 
-    public int getAlignment(int col) {
-        switch (col) {
+    public int getAlignment(final int column) {
+        switch (column) {
             case COL_PRICE:
             case COL_PERCENT:
+            case COL_DELIVERY:
```

Comment:
Kind of a moot point, I suppose, but do we really want the player to know ahead of time how long it'll take to get a particular unit delivered?

Reply:
I think it is like shipping, so... it makes sense to me.